nipype/interfaces/traits_extension.py

```diff
@@ -33,7 +33,7 @@
 from traits.api import BaseUnicode
 from traits.api import Unicode
 
-DictStrStr = traits.Dict(str, (bytes, str))
+DictStrStr = traits.Dict((bytes,str,unicode), (bytes,str,unicode))
```

Comment:
Does ```__future__``` or something else is changing the environ content? 
Because in any other context, the environ is all defined in str (=python3 bytes).